dev/com.ibm.ws.javamail/src/com/ibm/ws/javamail/internal/injection/MailSessionDefinitionInjectionBinding.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015 IBM Corporation and others.
+ * Copyright (c) 2015,2017,2021
```

Comment:
In documentation, if I'm not wrong, it says we only need to update second year as modification date. First year should stay as creation date.